pkg/auth/webapp/service2.go

```diff
@@ -54,7 +53,7 @@ type Service2 struct {
 }
 
 func (s *Service2) CreateSession(session *Session, redirectURI string) (*Result, error) {
-	if err := s.Sessions.Create(session); err != nil {
+	if err := s.Sessions.Save(session); err != nil {
```

Comment:
I am ok with the change, but i am curious to know why do we need it?

Reply:
I realized I should use `EntryPointPost` in `select_account.go`. After switching to using it, I do not need this change.